Assets/Scripts/Multiplayer/CoopCamera.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using UnityEngine;
+
+namespace Multiplayer
+{
+    public class CoopCamera : MonoBehaviour
+    {
+        public GameObject mainPlayer;
+        public GameObject coopPlayer;
+
+        public void Start()
+        {
+            if (Statics.gameMode == Statics.GameMode.SINGLEPLAYER)
+            {
+                this.gameObject.transform.parent = mainPlayer.transform;
+            }
+        }
+
+        public void Update()
+        {
+            if (Statics.gameMode != Statics.GameMode.LOCAL_MULTIPLAYER) return;
+            
+            SetCameraToMiddle();
+        }
+
+        private void SetCameraToMiddle()
```

Comment:
But does it also scale or preferably focus one player?
What happens if both walk away too far? Wouldn't you see neither of them?

Reply:
yep, they should stay together